.pipelines/azure-pipelines.yml

```diff
@@ -87,4 +87,4 @@ steps:
     MaxRetryAttempts: '5'
 
 - script: $(Build.SourcesDirectory)\upload-packages.cmd $(Build.SourcesDirectory)\out\packages $(NUGETORGAPIKEY)
-  displayName: 'Publish NuGet Packages'
+  displayName: 'Publish NuGet Packages'
```

Comment:
Did you make any whitespace changes here?

Reply:
It happened due to changed line ending. Resolved it with notepad :)